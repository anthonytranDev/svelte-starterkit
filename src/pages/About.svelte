<section data-testid="About">
  <h1>About</h1>
</section>
