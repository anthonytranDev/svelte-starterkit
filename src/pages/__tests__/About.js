import "@testing-library/jest-dom/extend-expect"
import { render } from "@testing-library/svelte"

import About from "../About.svelte"

describe("About", () => {
  it("contains the about title", () => {
    const { getByTestId } = render(About)

    expect(getByTestId("About")).toMatchSnapshot()
  })
})
