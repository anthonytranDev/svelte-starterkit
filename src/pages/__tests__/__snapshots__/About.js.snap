// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About contains the about title 1`] = `
<section
  data-testid="About"
>
  <h1>
    About
  </h1>
</section>
`;
